kube/src/lib.rs

```diff
@@ -88,10 +88,6 @@ assert_cfg!(
 pub mod api;
 pub mod client;
 pub mod config;
-#[deprecated(note = "Replaced by the kube-runtime crate", since = "0.38.0")]
```

Comment:
oh, whoops, decided not to do this library as it felt like the last straw thing to fix for it.
so i guess this is dependent on https://github.com/clux/kube-rs/pull/454

Reply:
that was merged.